v20.1/cockroachcloud-upgrade-policy.md

```diff
@@ -1,10 +1,91 @@
+
+- CockroachDB now creates files without read permissions for the "others" group. Sites that automate file management (e.g., log collection) using multiple user accounts now must ensure that the CockroachDB server and the management tools running on the same system are part of a shared unix group.
+
+- The [`GRANT`](../v20.1/grant.html) and [`REVOKE`](../v20.1/revoke.html) statements now require that the requesting user already have the target privileges themselves. For example, `GRANT SELECT ON t TO foo` requires that the requesting user already have the `SELECT` privilege on `t`.
+
+### Let ongoing bulk operations finish
+
+Make sure there are no [bulk imports](import.html) or [schema changes](online-schema-changes.html) in progress. These are complex operations that can increase the potential for unexpected behavior during an upgrade.</span>
+
+To check for ongoing imports or schema changes, use [`SHOW JOBS`](show-jobs.html#show-schema-changes) or check the [**Jobs** page](admin-ui-jobs-page.html) in the Admin UI.
+
+{{site.data.alerts.callout_info}}
+Once your cluster is running v20.1, any ongoing schema changes will stop making progress, but [`SHOW JOBS`](show-jobs.html) and the [**Jobs** page](admin-ui-jobs-page.html) in the Admin UI will show them as running until the upgrade has been finalized. During this time, it won't be possible to manipulate these schema changes via [`PAUSE JOB`](pause-job.html)/[`RESUME JOB`](resume-job.html)/[`CANCEL JOB`](cancel-job.html) statements. Once the upgrade has been finalized, these schema changes will run to completion. **Note that this behavior is specific to upgrades from 19.2 to 20.1. It does not apply to other upgrades.**
+{{site.data.alerts.end}}
+
+### Prevent new schema changes
+
+Once your cluster is running v20.1, new [schema changes](online-schema-changes.html) will be blocked and return an error, with the exception of [`CREATE TABLE`](create-table.html) statements without foreign key references and no-op schema change statements that use `IF NOT EXISTS`. Update your application or tooling to prevent disallowed schema changes during the upgrade process. **Note that this behavior is specific to upgrades from 19.2 to 20.1. It does not apply to other upgrades.**
+
+### Prevent changes to user privileges
+
+Once your cluster is running v20.1, [`GRANT`](grant.html) and [`REVOKE`](revoke.html) statements will be blocked and return and error. This is because privileges are stored with table metadata and, therefore, privilege changes are considered schema changes, from an internal perspective. Update your application or tooling to prevent privilege changes during the upgrade process. **Note that this behavior is specific to upgrades from 19.2 to 20.1. It does not apply to other upgrades.**
+
+## Step 4. Start the upgrade
+
+
 This page describes the upgrade policy for CockroachCloud.
```

Comment:
Should upgrade policy be moved? This doc seems focused on how to while the policy part is focused on, well, policy! Also, it looks like it needs an update. e.g. it says this:

```
### Rollback support

The default finalization period for each major upgrade is 7 days from upgrade completion.
```

Also this:

```
If you are a CockroachCloud Admin, you will receive an email notification for each major version release. The email will have instructions on how to opt in to have your clusters upgraded to the new version.
```

Reply:
+ 1 to upgrade policy being separated from these instructions. I feel that this tutorial could live as a child page under the "upgrade my cluster" with a header of "Upgrade to 20.1". Then as new releases become available they can be added as child pages with instructions specific to that version upgrade, since the breaking changes will always be different.

Reply:
I've broken the page into two, @emily-horing, one for the upgrade policy and one for the 20.1 upgrade instructions.